main.py

```diff
@@ -355,7 +357,7 @@ def launch_demo(args):
                 frame_to_save = cv_interface.frame
                 out.write(frame_to_save)
             total_time = time.time() - initial_time
-            fps = 1 / (total_time)
+            fps = np.round(1 / (total_time),1)
```

Comment:
todo inside the interface (when loging, it's usefull to have the full precision)